app/translations/en/validation.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  validation:
+    blank: This field is required
```

Comment:
I'm not 100% sure how overwriting translations will behave on deployment. Core module [defines en.validation.blank as well](https://github.com/Platform-OS/pos-module-core/blob/master/modules/core/public/translations/en/validation.yml#L6) - reasonable behavior to expect is that `app` should take precedence over modules/, but I am not sure if this is the case currently

Reply:
afaik these won't "overlap" as the namespace for module-specific translations are prefixed with the module name: `validation.blank` defined in `/modules/core/public/translations/en/validation.yml` will have `modules/core/validation.blank` as their translation key.
I think there are a few places in other modules where they don't specify the prefix so they rely on the main app to provide the right translation. It seems this is the case for the user module: https://github.com/Platform-OS/pos-module-user/blob/master/modules/user/public/views/partials/lib/commands/user/verify_password/check.liquid#L4-L6

Reply:
ahhh you are correct! so no conflict there - which was the point of the module prefix in the first place ;)